android/gradle.properties

```diff
@@ -1,3 +1,7 @@
-Apptentive_kotlinVersion=1.5.20
-Apptentive_compileSdkVersion=31
-Apptentive_targetSdkVersion=31
+Apptentive_kotlinVersion=1.7.20
+Apptentive_minSdkVersion=21
+Apptentive_compileSdkVersion=33
+Apptentive_targetSdkVersion=33
+Apptentive_ndkversion=21.4.7075529
```

Comment:
How do we get the ndkversion?

Reply:
It's actually unused. It was auto-generated. I can remove it.